packages/dds/map/src/test/mocha/map.spec.ts

```diff
@@ -773,6 +777,133 @@ describe("Map", () => {
 		});
 	});
 
+	describe("Attribution", () => {
+		describe("Local State", () => {
+			let map: SharedMap;
+
+			beforeEach(async () => {
+				const options: IMapOptions = { attribution: { track: true } };
+				map = createLocalMap("testMap", options);
+			});
+
+			it("Should not generate attribution in local state", () => {
+				map.set("first", "second");
+				map.set("third", "fourth");
+				map.set("fifth", "sixth");
+
+				const summaryContent = (map.getAttachSummary().summary.tree.header as ISummaryBlob)
+					.content;
+
+				assert.equal(
+					map.getAllAttribution()?.size,
+					0,
+					"do not have correct number of attribution",
+				);
+				assert.equal(
+					summaryContent,
+					`{"blobs":[],"content":{"first":{"type":"Plain","value":"second"},"third":{"type":"Plain","value":"fourth"},"fifth":{"type":"Plain","value":"sixth"}}}`,
+				);
+			});
+		});
+
+		describe("Connected State", () => {
+			let containerRuntimeFactory: MockContainerRuntimeFactory;
+			let map1: SharedMap;
+			let map2: SharedMap;
+
```

Comment:
We should have some tests here for `getAllAttribution` and `getAttribution` for really simple cases, e.g. single key set or double key set with no conflicts and checking that all clients agree. the test you have here is good and also provides nice coverage, but there are simpler things that could go wrong as well.

Reply:
resolved by 3114c8c35bd88abdae1eabe32b4835f725ee0847